ospfd/ospf_ri.c

```diff
@@ -1723,9 +1723,11 @@ DEFUN (router_info,
 
 DEFUN (no_router_info,
        no_router_info_cmd,
-       "no router-info",
+       "no router-info [<area|as>]",
```

Comment:
What if I enter `no router-info area 0.0.0.0`? Should we cover this case also?

Reply:
That command is deprecated to my tests, even if you set it `router-info area 0.0.0.0` vtysh will not report the area on `show run` - you only see `router-info area`

Reply:
oh, right...